desktop/libs/notebook/src/notebook/connectors/sql_alchemy_tests.py

```diff
@@ -219,6 +219,19 @@ def test_check_status(self):
       assert_equal(response['status'], 'available')
 
 
+  def test_explain(self):
+    snippet = Mock()
+    notebook = Mock()
+    with patch('notebook.connectors.sql_alchemy.SqlAlchemyApi.explain') as explain:
```

Comment:
This is not testing anything if you patch the method to test ;)

Instead do internal dependencies of the method like: `self._get_engine()` that way you can simulate the return of `engine.execute`